frontend/src/metabase/timelines/questions/components/TimelineEmptyState/TimelineEmptyState.tsx

```diff
@@ -27,20 +22,36 @@ const TimelineEmptyState = ({
     collection.can_write;
 
   const applicationName = useSelector(getApplicationName);
+
   return (
-    <EmptyStateRoot>
-      <EmptyStateIcon name="star" />
-      <EmptyStateText>
-        {canWrite
-          ? t`Add events to ${applicationName} to show helpful context alongside your data.`
-          : t`Events in ${applicationName} let you see helpful context alongside your data.`}
-      </EmptyStateText>
+    // The previous version of the empty state also had a hacky top margin set
+    // to arbitrary 10rem. This is because it is almost impossible to center this
+    // content vertically without touching element with test-id `sidebar-content`.
+    // Doing that would be risky because `SidebarContent` component is used by more
+    // than 15 other components at the moment.
+    //
+    // If we refactor the `TimelineSidebar` in the future, and decouple it from the
+    // `SidebarContent`, this empty state should be properly aligned vertically.
+    <Stack align="center" ta="center" gap="lg" mt="10rem">
+      <img src={EmptyEvent} alt={t`Collection event illustration`} />
+      <Box maw="25rem">
```

Comment:
What are the hardcoded widths here (and on the button below) for?

Reply:
They are the (new?) design specs. Pretty much the visual signature of these designs, actually.